### PR TITLE
Empty state integrations

### PR DESCRIPTION
**Pull request by thure**
This PR makes it easier for developers to supply a configuration for a Communication component to render in the primary surface if there is no content to display.

Currently only the List and Board components support this addition: the configuration for these components is extended to include an optional `emptyState` prop, which is the same as Communication’s props.

Communication was also adjusted to support multilingual `TTextObject` content.

## [Open in Storybook](https://dev-int.teams.microsoft.com/storybook/empty-states/index.html?path=/story/ui-templates-lists--empty)

### CODE REVIEW

**Review by akatrukhin — APPROVED** [review]
Before merging, could you improve logic for empty state. 
I don't think that it should be an option, that user could turn on/off.

**Comment by thure** [comment]
Transitioned back to draft since Board HVC needs integration as well.

**Comment by thure** [comment]
To do: use default content when `emptyState` is not supplied.

**Comment by thure** [comment]
This is now ready for review again.

**Review by akatrukhin — APPROVED** [review]
👍

File: src/components/List/List.tsx [hunk 1]
```diff
@@ -208,13 +214,22 @@ export const List = (props: IListProps) => {
...
+          {...tableProps}
+          {...{ onSelectedChange, filterBy }}
+          aria-live="polite"
+          id="fluentui-teams__list-content"
+          aria-label="List content"
+        />
+      ) : (
+        <Box styles={{ height: "calc(100vh - 4.25rem)" }}>
+          <Communication
+            {...props.emptyState}
```
[the remark this answers is not in the document]
Comment (thure) [reply src/components/List/List.tsx]: If the developer doesn't pass `emptyState`, then the List will simply be empty. To be fair, I'm not sure that's super helpful either… maybe a default empty state would be more helpful, but that may need to be written/designed.

**Review by asnow003 — APPROVED** [review]

**Pull request merged**